app/Http/Requests/AssetCheckoutRequest.php

```diff
@@ -14,6 +14,25 @@ public function authorize()
         return true;
     }
 
+    protected function prepareForValidation(): void
+    {
+        if (!empty($this->assigned_user)) {
+            $this->assigned_type =  'App\Models\User';
+        } elseif (!empty($this->assigned_asset)) {
+            $this->assigned_type =  'App\Models\Asset';
+        } elseif (!empty($this->assigned_location)) {
+            $this->assigned_type = 'App\Models\Location';
+        }
+
+        $this->replace([
+            'assigned_asset' => $this->assigned_asset,
+            'assigned_location' => $this->assigned_location,
+            'assigned_user' => $this->assigned_user,
```

Comment:
I'm used to seeing this as `assigned_to` - is there somewhere else where we transmogrify this variable that I'm missing?

Reply:
This is a form request, so we're only interpreting fields that are passed through `$request`, not the fields on the DB itself.